src/stream_to.cxx

```diff
@@ -34,6 +34,15 @@ void begin_copy(
         "COPY "sv, table, "("sv, columns, ") FROM STDIN"sv));
 }
 
+void begin_binary_copy(
+  pqxx::transaction_base &tx, std::string_view table, std::string_view columns)
+{
+  tx.exec0(
+    std::empty(columns) ?
+      pqxx::internal::concat("COPY "sv, table, " FROM STDIN WITH BINARY"sv) :
```

Comment:
Just a small note...  To be honest I'm not even sure at this point that this form ("copy all columns") is worth keeping.  Nowadays I feel that it's just way too sensitive to schema changes.  Consider that an historical mistake on my part.